lib/window.ts

```diff
@@ -41,7 +78,21 @@ export default function createWindow(currentWindow: Window, channel: Channel): W
 
   function updateHeight(height: number | null = null) {
     if (height === null) {
-      height = Math.ceil(document.documentElement.getBoundingClientRect().height)
+      const documentHeight = Math.ceil(document.documentElement.getBoundingClientRect().height)
+      const fullDocumentHeight = Math.max(
+        document.body.scrollHeight,
+        document.body.offsetHeight,
+        document.body.clientHeight,
+        document.documentElement.clientHeight,
+        document.documentElement.scrollHeight,
+        document.documentElement.offsetHeight
```

Comment:
`scrollHeight` <= `clientHeight` <= `offsetHeight`
Why use all three of them? Any situation where this is not the case?

https://medium.com/@jbbpatel94/difference-between-offsetheight-clientheight-and-scrollheight-cfea5c196937